subprojects/language-java/src/integTest/groovy/org/gradle/java/compile/incremental/SourceIncrementalJavaCompilationIntegrationTest.groovy

```diff
@@ -257,6 +257,8 @@ class SourceIncrementalJavaCompilationIntegrationTest extends AbstractIntegratio
         outputs.recompiledClasses 'B', 'A'
     }
 
+    @NotYetImplemented
+    // used to work in 3.4, not anymore in 3.4.1. Can re-enable with compiler plugins. See gradle/gradle#1474
     def "changed class with unused non-private constant incurs partial rebuild"() {
```

Comment:
Same as above, this method name is misleading. It's not just "any change", but changing the unused constant (removing in this case).